src/main/java/org/spongepowered/api/item/recipe/SmeltingRegistry.java

```diff
@@ -0,0 +1,37 @@
+package org.spongepowered.api.item.recipe;
+
+import org.spongepowered.api.item.inventory.ItemStack;
+
+/**
+ * A RecipeRegistry holds all registered smelting recipes for a given game.
+ */
+public interface SmeltingRegistry {
+    /**
+     * Adds a smelting recipe to this registry.
+     * @param input the input of the recipe
```

Comment:
Needs an empty javadoc line before this line. Same goes for all methods in this class.
Also the param descriptions should start with a capital letter after the param name.